packages/app-runtime/src/multiAccount/AccountServices.ts

```diff
@@ -38,6 +38,10 @@ export class AccountServices {
         return LocalAccountMapper.toLocalAccountDTO(localAccount);
     }
 
+    public async offboardAccount(id: string): Promise<void> {
```

Comment:
maybe we should just call this offboardDevice :)

Reply:
That's how I initially named it. Then, I saw that all methods have `Account` in their name, also `onboardAccount`. But I would be fine renaming it. Should `onboardAccount` also be renamed?

Reply:
As mentioned in my other comment, I would prefer to name the MultiAccountController method offboardAccount too, then everything matches.

Reply:
Should the MultiAccountController method `onboardDevice` be renamed to `onboardAccount`, too?

Reply:
argh, I hate this. I think we should leave the name as it triggers a deviceOnboarding which creates a new account..